fw/http.c

```diff
@@ -6710,6 +6712,7 @@ tfw_http_resp_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb)
 	 * protocol data.
 	 */
 	if (skb && !websocket) {
+		*splitted = NULL;
 		hmsib = tfw_http_msg_create_sibling(hmresp, skb);
```

Comment:
pay attention here. If we successfully create `hmsib` but fails later. Who frees `hmsib`?